packages/compiler-ssr/__tests__/ssrComponent.spec.ts

```diff
@@ -16,6 +16,27 @@ describe('ssr: components', () => {
       }"
     `)
   })
+  
+  test("multiple _ssrInterpolate at parent and child import dependency once", () => {
+      expect( compile(`<div>{{ hello }}<textarea v-bind="a"></textarea></div>`).code)
+      .toMatchInlineSnapshot(`
+        "const { ssrRenderAttrs: _ssrRenderAttrs, ssrInterpolate: _ssrInterpolate } = require(\\"@vue/server-renderer\\")
+
+        return function ssrRender(_ctx, _push, _parent, _attrs) {
+          let _temp0
+
+          _push(\`<div\${
+            _ssrRenderAttrs(_attrs)
+          }>\${
+            _ssrInterpolate(_ctx.hello)
+          }<textarea\${
+            _ssrRenderAttrs(_temp0 = _ctx.a, \\"textarea\\")
+          }>\${
+            _ssrInterpolate((\\"value\\" in _temp0) ? _temp0.value : \\"\\")
+          }</textarea></div>\`)
+        }"
+      `);
+    });
```

Comment:
The test case should be written in `ssrElement.spec.ts`

Reply:
moved test case to ssrElements.spec.ts,  
do you want to suggest a different test name